tests/integration-tests/tests/networking/test_networking.py

```diff
@@ -50,13 +50,17 @@ def vpc_stack(vpc_stacks, region):
     return vpc_stacks[region]
 
 
+@pytest.mark.regions(["eu-central-1", "us-gov-east-1", "cn-northwest-1"])
 def test_public_network_topology(region, vpc_stack, networking_stack_factory):
     ec2_client = boto3.client("ec2", region_name=region)
     vpc_id = vpc_stack.cfn_outputs["VpcId"]
     public_subnet_cidr = "10.0.3.0/24"
     availability_zone = AVAILABILITY_ZONE_OVERRIDES.get(region, "")
```

Comment:
You could have done `random.choice(AVAILABILITY_ZONE_OVERRIDES.get(region, [None]))` so that you randomly pick an AZ and you don't need the if below